clang/lib/Frontend/VerifyDiagnosticConsumer.cpp

```diff
@@ -396,6 +396,15 @@ class VerifyDiagnosticConsumer::MarkerTracker {
   }
 };
 
+std::string DetailedErrorString(DiagnosticsEngine &Diags) {
+  std::string ErrDirective;
+  if (Diags.getDiagnosticOptions().VerifyPrefixes.empty())
+    ErrDirective = "expected";
+  else
+    ErrDirective = *Diags.getDiagnosticOptions().VerifyPrefixes.begin();
+  return ErrDirective;
+}
```

Comment:
```suggestion
static std::string DetailedErrorString(const DiagnosticsEngine &Diags) {
  if (Diags.getDiagnosticOptions().VerifyPrefixes.empty())
    return "expected";
  return *Diags.getDiagnosticOptions().VerifyPrefixes.begin();
}
```
Some minor changes here but are mostly NFC.

Reply:
Done